pageserver/src/tenant/storage_layer/layer.rs

```diff
@@ -604,9 +604,17 @@ enum Status {
 
 impl Drop for LayerInner {
     fn drop(&mut self) {
+        // if there was a pending cancellation, mark it cancelled here to balance metrics
```

Comment:
Did you mean `pending eviction`?

Reply:
yes good catch

Reply:
[b837401](https://github.com/neondatabase/neon/pull/7363/commits/b837401fb09d2d9818b70e630fdb67e9799b7b0d)